MapboxNavigation/StyleManager.swift

```diff
@@ -29,17 +29,40 @@ public protocol StyleManagerDelegate: NSObjectProtocol {
 @objc(MBStyleManager)
 open class StyleManager: NSObject {
     
-    weak var delegate: StyleManagerDelegate?
+    /**
+     The receiver of the delegate. See `StyleManagerDelegate` for more information.
+     */
+    @objc public weak var delegate: StyleManagerDelegate?
     
-    internal var date: Date?
-    var currentStyleType: StyleType?
-    var styles = [Style]() { didSet { applyStyle() } }
-    var automaticallyAdjustsStyleForTimeOfDay = true {
+    /**
+     Determines whether the style manager should apply a new style given the time of day.
+     
+     - note: Two styles must be provided for this property to have any effect.
```

Comment:
Nit: this doesn’t rise to the level of a `note`. `precondition` seems more appropriate, in any case.